### PR TITLE
[FSPE-5417] Improved MobileNav dispose method

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

File: src/chi/javascript/components/mobile-navigation.js [hunk 1]
```diff
@@ -110,8 +111,9 @@ class MobileNavigation extends Component {
       firstLevelDropdownTriggers,
       (dropdown) => {
         const dropdownElemId = dropdown.getAttribute('data-target');
+        const dropdownInstant = Dropdown.factory(dropdown, { dropdownElem: document.querySelector(`#${dropdownElemId}`) });
 
-        Dropdown.factory(dropdown, { dropdownElem: document.querySelector(`#${dropdownElemId}`) });
+        this._dropdowns.push(dropdownInstant);
```
Comment (dani-cl-madrid) [inline src/chi/javascript/components/mobile-navigation.js:116]: 👏👏👏👏👏

File: src/chi/javascript/components/mobile-navigation.js [hunk 2]
```diff
@@ -441,18 +445,21 @@ class MobileNavigation extends Component {
   }
 
   dispose() {
+    this._firstLevelDrawer.dispose();
     this._config = null;
     this._elem = null;
-    this._firstLevelDrawer = null;
```
Comment (dani-cl-madrid) [inline src/chi/javascript/components/mobile-navigation.js:446]: Please keep this line. First call dispose, as you're currently doing, and then, set it to null or delete it. E.g.
```Javascript
delete this._firstLevelDrawer;
```

**Review by dani-cl-madrid — CHANGES_REQUESTED** [review]
Just one minor change.

File: src/chi/javascript/components/mobile-navigation.js (hunk 2 above)
Comment (janibolkvadze) [reply src/chi/javascript/components/mobile-navigation.js:446]: Done, thank you!

**Review by dani-cl-madrid — APPROVED** [review]

**Pull request merged**